spec/generator/modules_generator_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Modules', type: :generator do
```

Comment:
should this file be removed since the tests are in `spec/lib/generators/module/module_generator_spec.rb`?

Reply:
Yes! Thanks for pointing this out